src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/MS/Internal/Tasks/TaskHelper.cs

```diff
@@ -212,17 +212,17 @@ internal static bool IsValidCultureName(string name)
             return bValid;
         }
 
-#if false
+#if NETFX 
```

Comment:
Please add a comment near `#if NETFX` indicating that it continues to be useful in the .NET Core 3.0+ codebase, and exists intentionally. 

We are in the process of ripping out `#if NETFX` from the codebase and most of those are _TODO's_ or "remove this section" type conditionals. Having a nearby comment would reduce confusion during refactoring/cleanup.

Reply:
At this point this comment is probably irrelevant. All the _todo_ style `#if NETFX` entries have been removed from the codebase. rebase or merge with master and you should be good to go. Commenting is probably still a good idea for documentation.